atuin-client/src/settings.rs

```diff
@@ -55,6 +55,7 @@ impl FilterMode {
 }
 
 // FIXME: Can use upstream Dialect enum if https://github.com/stevedonovan/chrono-english/pull/16 is merged
+// FIXME: Above PR was merged, but dependency was changed to interim (fork of chrono-english) in the ... interim
```

Comment:
:D

Reply:
This actually is still necessary because of the serde usage. The chrono-english PR doesn't add this serde feature so it's not enough. However I have ways to avoid a new type, which I can do some point later

Reply:
Should I just remove both comments, then ?

Reply:
Nah it's amusing, keep it